src/org/usfirst/frc/team1076/robot/commands/DriveLeftRight.java

```diff
@@ -0,0 +1,53 @@
+package org.usfirst.frc.team1076.robot.commands;
+
+import org.usfirst.frc.team1076.robot.subsystems.FrontBackMotors;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class DriveLeftRight extends Command {
+
+	FrontBackMotors frontBack;
+	double speed;
+	double time;
+	double targetTime;
+
+	public DriveLeftRight(FrontBackMotors frontBack, double targetTime, double speed) {
+		this.frontBack = frontBack;
+		this.speed = speed;
+		this.targetTime = targetTime;
+		requires(frontBack);
+	}
+
+	// Called just before this Command runs the first time
+	protected void initialize() {
+		this.time = 0;
+	}
+
+	// Called repeatedly when this Command is scheduled to run
+	protected void execute() {
+		time += 1 / 50.0;
+		frontBack.setSpeed(speed);
+	}
+
+	// Make this return true when this Command no longer needs to run execute()
+	protected boolean isFinished() {
+		if (time == targetTime) {
```

Comment:
If the robot is too close to the target distance when it checks isFinished that it overshoots at moves too far by the time isFinished is checked again, so time is bigger than target time, the robot won't stop.` time>=target time ` will fix this bug.